templates/jinja2/index.html

```diff
@@ -96,13 +91,19 @@ <h1>Search Illinois public employee salary data</h1>
       <div class="col-md">
         <h3>What's inside?</h3>
 
-        <p>The <strong>Illinois Public Salaries Database</strong> contains the base and additional pay (such as overtime) for <strong>{{ salary_count|format_exact_number }}</strong> state, municipal, school and many other government employees across Illinois. It covers <strong>{{ unit_count|format_exact_number }}</strong> units of government. We’ve recently updated the database so users find it easier to navigate. <a href="{{ url('user_guide') }}">You can learn how to use this new database here</a>.</p>
+        <p>The <strong>Illinois Public Salaries Database</strong> contains the base and additional pay (such as overtime) for <strong id='index-salary-count'></strong> state, municipal, school and many other government employees across Illinois. It covers <strong id='index-unit-count'></strong> units of government. We’ve recently updated the database so users find it easier to navigate. <a href="{{ url('user_guide') }}">You can learn how to use this new database here</a>.</p>
       </div>
     </div>
 
     <div class="row mb-3">
       <div class="col-md">
-        <h5>Distribution of public salaries in {{ data_year }}</h5>
+        <h5><label for="index-data-year">Distribution of public salaries in </label>
+          <select id="index-data-year">
+            <option>Select Year</option>
+            <option value="2017">2017</option>
+            <option value="2018">2018</option>
```

Comment:
It would be ideal to generate the list of years dynamically, so this will update automatically as data is added.

Reply:
I'd suggest adding a variable in `context_processors.py` equal to a list of distinct reporting years. You can get that list by querying  `StandardizedFile`, i.e., `StandardizedFile.objects.distinct('reporting_year').values('reporting_year')`